packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -126,6 +126,16 @@ export interface PipelineProps {
    * @default true
    */
   readonly crossAccountKeys?: boolean;
+
+  /**
+   * Enables KMS key rotation for cross-account keys
```

Comment:
This is not only for the cross-account keys, but for all keys, right? Including the main artifact store key if the pipeline is not cross-region.

Let's update this doc to reflect that (and also let's end the sentence with a period).